### PR TITLE
DBE-2717 Change information_schema syntax to support MySQL 5x

### PR DESCRIPTION
**Pull request by mcstafford-git**
- **DBE-2717 Change information_schema syntax to support MySQL 5x**
- **DBE-2717 Add actions**

### CODE REVIEW

File: cmd/sqlpipe/system_mysql.go [hunk 1]
```diff
@@ -742,7 +742,16 @@ func (system Mysql) getSqlFormatters() (
 
 func (system Mysql) getTableColumnInfosRows(schema, table string) (rows *sql.Rows, err error) {
 	query := fmt.Sprintf(`
-		WITH PrimaryKeys AS (
+		SELECT
```
Comment (ChaosHour) [inline cmd/sqlpipe/system_mysql.go:745]: Missing         SELECT DISTINCT

Comment (mcstafford-git) [reply cmd/sqlpipe/system_mysql.go:745]: When the column is grouped in the subquery it doesn't need distinct.

Comment (ChaosHour) [reply cmd/sqlpipe/system_mysql.go:745]: I ran into an issue in the results where that DISTINCT was needed.   I guess you can try it out and see for yourself?

Comment (mcstafford-git) [reply cmd/sqlpipe/system_mysql.go:745]: Yes, when done with out the grouping that would be the case.

I included grouping.

Comment (ChaosHour) [reply cmd/sqlpipe/system_mysql.go:745]: On the table I was testing on, it was returning duplicate column names in the result ... messing up the destination table.

**Review by ChaosHour — APPROVED** [review]

File: cmd/sqlpipe/system_mysql.go (hunk 1 above)
Comment (ChaosHour) [reply cmd/sqlpipe/system_mysql.go:745]: You are correct with using of the grouping.  Good call.

Comment (mcstafford-git) [reply cmd/sqlpipe/system_mysql.go:745]: Thank you for the attentive review.

**Pull request merged**